install_config/aggregate_logging.adoc

```diff
@@ -724,6 +724,10 @@ named to match the `openshift_logging_es_pvc_prefix` setting, which defaults to
 exists already, it is used; if not, and `openshift_logging_es_pvc_size` has been
 specified, it is created with a request for that size.
 
+If using NFS storage, the {product-title} installer creates the persistent volumes, based on the `openshift_logging_storage_*` parameters 
+and the xref:#deploying-the-efk-stack[{product-title} logging installer] creates PVCs, using the openshift_logging_es_pvc_ paramters.
+Make sure you specify the correct parameters in order to use persistent volumes with EFK.
```

Comment:
Maybe add something above this like:
"If using xref:install_config/persistent_storage/dynamically_provisioning_pvs.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[dynamically provisioned PVs] then {product-title} logging installer will create PVCs that will use the default storage class or the one specified with the `openshift_logging_elasticsearch_pvc_storage_class_name` parameter.